cumulusci/cli/tests/test_org.py

```diff
@@ -801,7 +802,13 @@ def test_org_prune(self, echo):
             "active2",
             "persistent",
         ]
-        runtime.project_config.orgs__scratch = {"shape1": True, "shape2": True}
+
+        runtime.project_config.lookup = MockLookup(
+            orgs__scratch={
+                "shape1": True,
+                "shape2": True,
+            },
+        )
```

Comment:
I'd strongly encourage rewriting the tests to use real runtime, keychain, and project config objects instead of mocks. I worked on this for 1 or 2 of the CLI modules already (the same ones that got updated to use run_cli_command instead of run_click_command) but didn't get to this one yet.

Reply:
Okay but that would be a separate code cleanup I guess?